### PR TITLE
On-premises SOAR end-of-life release note

### PR DESCRIPTION
**Pull request by jpipkin1**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request adds a release note announcing end-of-life for the SOAR on-premises solution. We'll place the release note here: https://help.sumologic.com/release-notes-csoar/

Issue: Asana ticket "[Write release note for SOAR on-premises end-of-life](https://app.asana.com/0/1183741177809740/1205848521982246/f)"

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
@stacykor, this PR is in draft until I get your feedback.

**Comment by jpipkin1** [comment]
This PR is now ready for review.

File: blog-csoar/2023-11-01-23-application-update.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+image: https://help.sumologic.com/img/sumo-square.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+### Sumo Logic On-Premises SOAR Solution End-of-Life
+
+As of **November 15, 2023**, Sumo Logic's on-premises SOAR solution no longer receives updates, and Sumo Logic Engineering no longer develops, repairs, maintains, or tests the software. 
+
+On **December 1, 2024**, Sumo Logic’s on-premises SOAR solution reaches its end-of-life and is obsolete. Beginning on that date, it no longer receives applicable support entitled by active support contracts or by applicable warranty terms and conditions.
```
Comment (kimsauce) [inline blog-csoar/2023-11-01-23-application-update.md:18]:
```suggestion
Effective **December 1, 2024**, Sumo Logic’s on-premises SOAR solution reaches end-of-life and becomes obsolete. Beginning on that date, it no longer receives applicable support entitled by active support contracts or by applicable warranty terms and conditions. 
```

File: blog-csoar/2023-11-01-23-application-update.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+### Sumo Logic On-Premises SOAR Solution End-of-Life
+
+As of **November 15, 2023**, Sumo Logic's on-premises SOAR solution no longer receives updates, and Sumo Logic Engineering no longer develops, repairs, maintains, or tests the software. 
+
+On **December 1, 2024**, Sumo Logic’s on-premises SOAR solution reaches its end-of-life and is obsolete. Beginning on that date, it no longer receives applicable support entitled by active support contracts or by applicable warranty terms and conditions. 
+
+To continue to utilize Sumo Logic’s SOAR capabilities, use our SAAS product, [Cloud SOAR](/docs/cloud-soar/).
```
Comment (kimsauce) [inline blog-csoar/2023-11-01-23-application-update.md:20]:
```suggestion
To continue to utilize Sumo Logic’s SOAR capabilities, use our SaaS product, [Cloud SOAR](/docs/cloud-soar/). 
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
couple suggestions!

**Comment by jpipkin1** [comment]
Good suggestions, @kimsauce!

**Comment by jpipkin1** [comment]
@kimsauce, could you approve this PR?

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
Thanks, @kimsauce!

**Pull request merged**